### PR TITLE
Regen

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gir/pull/822.

cc @EPashkin @sdroege

### CODE REVIEW

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks

**Comment by GuillaumeGomez** [comment]
Hmmm... Still failing but this time it's a bit strange. I'll need to check that.

**Comment by EPashkin** [comment]
@GuillaumeGomez CI fails because gtk-sys not regened.

File: gtk-sys/Cargo.toml [hunk 1]
```diff
@@ -33,6 +33,7 @@ shell-words = "0.1.0"
 tempdir = "0.3"
 
 [features]
+docs-rs = []
```
Comment (EPashkin) [inline gtk-sys/Cargo.toml]: @GuillaumeGomez can you remove this feature?

Comment (GuillaumeGomez) [reply gtk-sys/Cargo.toml]: Sure.

**Comment by EPashkin** [comment]
Thanks, and 👍

**Comment by GuillaumeGomez** [comment]
Sounds like this time we're good! Thanks for your review @EPashkin !

**Pull request merged**